test/write.spec.ts

```diff
@@ -162,7 +165,8 @@ describe('writeBenchmark()', function () {
         name: 'User',
         username: 'user',
     };
-    const repoUrl = 'https://github.com/user/repo';
+    const serverUrl = 'https://github.com';
+    const repoUrl = `${serverUrl}/user/repo`;
```

Comment:
with `describe.each` approach you'll need only the `repoUrl` variable here
```suggestion
    const repoUrl = `${serverUrl}/user/repo`;
```